<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(245,222,179)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background:white</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Patient Info</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_patientId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_patientName">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_city">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_age">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>290</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phone">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fathername">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>210</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_gender">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_doctor">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>280</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_add_patient">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>430</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Patient Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>City</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>40</y>
         <width>91</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>FatherName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Phone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Doctor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Doctor Info</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_doctorId">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_doctorName">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_doctorPhone">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>50</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_doctorEmail">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>120</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dutyTime">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>200</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_specialization">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Pediatricians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dentist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Geriatricians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dermatologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Opthalmologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neurologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General Surgeons</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orthopaedic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cardiologists</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_department">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>271</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Emergency Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teeth Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eye Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Genriatric Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cardio Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orthopaedic Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plastic Surgery Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Paediatric Medical Ward</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Birth Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neurology Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infectious Diseases </string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_degree">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>340</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Ph.D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M.D</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_add_doctor">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>430</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>111</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Doctor Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Specialization</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Department</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Degree</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>30</y>
         <width>111</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Phone No</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Duty Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Doctor List</string>
      </attribute>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Department</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Specialization</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_department_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Emergency Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Teeth Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eye Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Genriatric Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Carido Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orthopaedic Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plastic Surgery Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Paediatric Medical Ward</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Birth Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neurology Department</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infectious Diseases</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_specialization_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Pediatricians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dentist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Geriatricians</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dermatologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Opthalmologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neurologists</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General Surgeons</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orthopaedic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cardiologists</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="tableView_doctor_list">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>721</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_show">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>160</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
